### DISCUSSION TITLE
Representation of cash in portfolio

### DISCUSSION THREAD

**Discussion by indusidler** [OpenBB-finance/OpenBB#5240] [Q&A]
hi,
Trying to upload a portfolio... I am able to get the file to the required format. The quantities I believe should be absolute values. However how do I represent cash? I have put the ticker as USD but when I try to do a holdp, I dont see the cash component being there. Also a bunch of names are showing up in negative percentages even though I dont have any short position at all.

**Comment by markoblogo** [comment]
When you upload a portfolio to OpenBB, cash is not treated as a regular stock ticker like USD, so it will not appear as a separate line item in holdp the way equities do. Instead, the tool focuses on the securities and computes their weights from the positions you provide, which can lead to some names showing negative percentage weights if your input quantities or the initial portfolio value make the allocation calculation unbalanced.
A practical workaround is to keep your cash outside the uploaded holdings and treat it as the residual between your total account value and the market value of the securities in the OpenBB portfolio, rather than forcing it in as a ticker. If you want to double‑check how your file is being interpreted, you can share a minimal anonymized sample of the CSV (just a couple of lines with tickers, quantities, and prices), and the community can help spot why some weights show up as negative even though you do not have short positions.
